homeassistant/components/upnp/coordinator.py

```diff
@@ -13,7 +13,7 @@
 from .device import Device
 
 
-class UpnpDataUpdateCoordinator(DataUpdateCoordinator):
+class UpnpDataUpdateCoordinator(DataUpdateCoordinator[dict[str, Any]]):
```

Comment:
I don't see how you use a `dict` with your `DataUpdateCoordinator`.

Reply:
The `UpnpDataUpdateCoordinator._async_update_data()` does return a `dict[str, Any]`. The result of that method is stored into `UpnpDataUpdateCoordinator`/`DataUpdateCoordinator` `.data`.

I also see other components doing this as well.

Reply:
I've made `Any` more specific in commit https://github.com/home-assistant/core/pull/89954/commits/e1f12da969c3213479845dd4aa00210099d39c2d.

Reply:
Sorry for the errant comment.  Guess I have more to learn about Typing.

Reply:
No problem. This made me be more specific, i.e., replace the `Any` with the actual types.